packages/storybook-addon/src/decorator/react.ts

```diff
@@ -4,13 +4,22 @@ import { createElement } from 'react';
 import { Provider, createClient } from 'urql';
 import { getStorybookExchange } from '../exchange';
 
+/* eslint-disable */
+const { devtoolsExchange } = require('@urql/devtools');
```

Comment:
I see what the intention here is, e.g. along the lines of `try { require } catch (_e) {}` but since all decorators/addons run only on the client-side this will be going through Webpack, which will likely complain about missing dependencies. Not sure if we have an example repository to test this out on.

I think it's perfectly fine just to make our storybook addon depend on the devtools exchange and to then add it explicitly :+1: